src/Button/__tests__/BorderlessButton.test.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { render, fireEvent, cleanup } from 'test-utils';
+import { spy } from 'sinon';
+import assert from 'assert';
+import BorderlessButton from '../BorderlessButton';
+
+describe('Borderless Button', function () {
+  let getByText;
+  let clickSpy;
+  beforeEach(function () {
+    clickSpy = spy();
+    ({ getByText } = render(
+      <BorderlessButton clickHandler={clickSpy}>
+        Click Me
+      </BorderlessButton>
+    ));
+  });
+  afterEach(function () {
+    clickSpy.resetHistory();
+    cleanup();
```

Comment:
You shouldn't need to call `cleanup()` in each test, since it's being done in the `.mochainit.ts` file in the project root.

_This applies to other button tests as well._

Reply:
Thanks! Fixed through d6fad8a.

Reply:
Ah. I forgot that there are some issues with running tests in `watch` mode related to this. I think I've found a possible solution (clearing the node `require` cache in the `afterEach` block). I'll submit that as a separate PR